sgv2-dynamo-api/src/main/java/io/stargate/sgv2/dynamosvc/parser/ProjectionExpressionVisitor.java

```diff
@@ -0,0 +1,50 @@
+package io.stargate.sgv2.dynamosvc.parser;
+
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import java.util.Map;
+import org.antlr.v4.runtime.tree.ParseTree;
+
+public class ProjectionExpressionVisitor extends ProjectionBaseVisitor<Object> {
+
+  private Map<String, String> nameMap;
```

Comment:
Seems like this and next field could be `final`?

Reply:
`nameMap` can be final, but `item` cannot. The tricky part is in the `visitMapAttr` method, where we assign the child branch to `item`. Not sure if there is a better way to deal with it.

Reply:
Right, I thought that `item` could be made `final` by rewriting the handling, but perhaps that's not worth it (or not possible).